### PR TITLE
Added django 2.0 support

### PR DESCRIPTION
**Pull request by jamstooks**
I added travis config file for passing tests: https://travis-ci.org/jamstooks/django-membersuite-auth.

You'll need to add these environment vars to your travis config:

    - `MS_ACCESS_KEY`
    - `MS_SECRET_KEY`
    - `MS_ASSOCIATION_ID`
    - `TEST_MS_PORTAL_USER_ID`
    - `TEST_MS_PORTAL_USER_PASS`
    - `TEST_MS_PORTAL_USER_FIRST_NAME`
    - `TEST_MS_PORTAL_USER_LAST_NAME`

### CODE REVIEW

**Comment by jamstooks** [comment]
You'll see I dropped the pip-compile stuff. It seemed like overkill for this project. Of course, if there's a reason I should put it back, let me know and I can do that.

File: django_membersuite_auth/tests.py [hunk 1]
```diff
@@ -12,8 +12,10 @@
 from .services import MemberSuitePortalUserService
 
 
-TEST_MS_PORTAL_USER_ID = os.environ["TEST_MS_PORTAL_USER_ID"]
-TEST_MS_PORTAL_USER_PASS = os.environ["TEST_MS_PORTAL_USER_PASS"]
+TEST_USER_ID = os.environ["TEST_MS_PORTAL_USER_ID"]
```
Comment (rerb) [inline django_membersuite_auth/tests.py:15]: This mismatch between Python variable name and environmental variable name is something we'll have to fix.  I'll open a ticket.

File: django_membersuite_auth/models.py [hunk 2]
```diff
@@ -8,7 +8,8 @@
 @python_2_unicode_compatible
 class MemberSuitePortalUser(models.Model):
 
-    user = models.OneToOneField(settings.AUTH_USER_MODEL)
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```
Comment (rerb) [inline django_membersuite_auth/models.py:12]: Is this cool?  Do we really want, say, resources added to the hub by a user to be deleted when a MemberSuitePortalUser is deleted?

**Review by rerb — APPROVED** [review]
Hey ben -- sure about that cascading delete on MemberSuitePortalUser?

**Pull request merged**